juriscraper/opinions/united_states/state/mdag.py

```diff
@@ -1,165 +1,53 @@
-        return urls
-
-    def _get_precedential_statuses(self) -> list:
-        statuses = []
-        path = self.cell_path % 1
-        for tree in self.html:
-            for cell in tree.xpath(path):
-                if "Unpublished" in cell.text_content():
-                    statuses.append("Unpublished")
-                else:
-                    statuses.append("Published")
-        return statuses
-
-    def _set_date_properties(self, today: str = None) -> None:
-        """Pass in artificial 'today' option for testing only."""
-        self.today = convert_date_string(today) if today else date.today()
-        self.year = self.today.year
-
-    def get_dynamic_html_trees(self) -> list:
-        self.initiate_webdriven_session()
-
-        # Find and activate the opinion drop-down for year
-        try:
-            path = f"{self.parent_path}/a"
-            date_anchor = self.find_element_by_xpath(path)
-        except NoSuchElementException:
-            # Year has no opinions drop-down on page
-            return []
-
-        # if subscription overlay dialog div is present,
-        # close it, we will not be subscribing :)
-        path_dialog = "//*[@title='Close subscription dialog']"
-        dialog = self.find_element_by_xpath(path_dialog)
-        if dialog:
-            dialog.click()
-
-        # click the appropriate date anchor
-        date_anchor.click()
-
-        trees = [self.get_tree_from_driver_dom()]
-
-        # Handle pagination if more than 30 results for year
-        while True:
-            try:
-                next_anchor = self.find_element_by_xpath(self.next_path)
-            except NoSuchElementException:
-                # Less than 30 results
-                break
-            next_anchor.click()
-            trees.append(self.get_tree_from_driver_dom())
-        return trees
-
-    def get_tree_from_driver_dom(self) -> list:
-        # Wait for js to load and dom html to update
-        # Seems stupid, but necessary, and easier
-        # thank loading lots of selenium dependencies
-        # and using complex WebDriverWait with callbacks
-        # for attribute to appear, which don't even
-        # seem to work consistently with the site's
-        # finicky responses.
-        sleep(3)
-        script = "return document.getElementsByTagName('html')[0].innerHTML"
-        source = self.webdriver.execute_script(script)
-        tree = html.fromstring(source)
-        tree.make_links_absolute(self.domain)
-        return tree
+    def _process_html(self):
+        for row in self.json["Row"]:
+            url_path = row["FileRef.urlencodeasurl"]
+            self.cases.append(
+                {
+                    "docket": row["Title"],
+                    "summary": row["Summary"],
+                    "name": row["FileLeafRef.Name"],
+                    "url": f"https://www.marylandattorneygeneral.gov{url_path}",
+                    "date": row["Created_x0020_Date"][3:13],
```

Comment:
Can you give an example of what `3:13` is handling, please?

Reply:
OK, so seeing that example (which I'd expect above the `"date"` line), not above the `append()`, I wonder if we should grab the date by regex or by splitting on the `#`, etc. I just worry that too much can change in that random string at the beginning.